node/src/coordinator.rs

```diff
@@ -382,9 +382,24 @@ impl Coordinator {
     ) -> anyhow::Result<MpcJobResult> {
         tracing::info!("Entering running state.");
         let keyshare_storage = Arc::new(keyshare_storage);
-
-        delete_stale_triples_and_presignatures(&secret_db, running_state.keyset.epoch_id)?;
-
+        let my_participant_id = running_state
+            .participants
+            .get_participant_id(&config_file.my_near_account_id);
+        if let Some(my_participant_id) = my_participant_id {
+            let current_participants_config = running_state.participants.clone();
+            let current_epoch_id = running_state.keyset.epoch_id;
+            let all_domains: Vec<DomainId> = running_state.keyset.get_domain_ids();
+            let current_epoch_data = EpochData {
+                epoch_id: current_epoch_id,
+                participants: current_participants_config,
+            };
+            delete_stale_triples_and_presignatures(
+                &secret_db,
+                current_epoch_data,
+                my_participant_id,
+                all_domains,
+            )?;
+        }
```

Comment:
Does this mean we only run this when we enter the running state? What if the nodes are running for a long time and one participant updates their state?

Reply:
Got clarification on slack. The purpose of https://github.com/near/mpc/pull/1032 is to handle that.